### ISSUE TITLE
Feature request: conceal checkboxes

### ISSUE THREAD

**Issue by Alok** [vim-pandoc/vim-pandoc-syntax#203]
Or add an option to do so. I realize that checkboxes like GFM supports are not in Pandoc Markdown, but they're damned useful.

**Comment by alok** [comment]
Now that pandoc officially supports checkboxes, I was wondering if this feature could be considered.

**Comment by alerque** [comment]
Yes @alok this would be considered. If anybody wants to put together the rules and submit PR I'd be happy to review and merge it.
